react/features/mobile/external-api/middleware.ts

```diff
@@ -253,7 +256,7 @@ externalAPIEnabled && StateListenerRegistry.register(
     /* listener */ debounce((tracks: ITrack[], store: IStore) => {
         const oldScreenShares = store.getState()['features/mobile/external-api'].screenShares || [];
         const newScreenShares = tracks
-            .filter(track => track.mediaType === 'video' && track.videoType === 'desktop')
+            .filter(track => track.mediaType === MEDIA_TYPE.SCREENSHARE && track.videoType === VIDEO_TYPE.DESKTOP)
             .map(track => track.participantId);
```

Comment:
Why don't you use some of the utilities we have? 
https://github.com/jitsi/jitsi-meet/blob/bb7ae777b0194efc7c9bc1dc9cb6cf477c7abdcc/react/features/base/tracks/functions.any.ts#L247C23-L247C23